input/cql/PCSBMIScreenAndFollowUpFHIR.cql

```diff
@@ -113,59 +101,50 @@ define "High BMI Interventions Ordered":
         )
   )
 
-/*
-@QICore6Uplift: Update 'Condition to 'ConditionProblemsHealthConcerns'
-*/
 define "High BMI Interventions Performed":
   ( [Procedure: "Follow Up for Above Normal BMI"] HighInterventionsPerformed
       where HighInterventionsPerformed.reasonCode in "Overweight or Obese"
-        or ( exists ["Condition": "Overweight or Obese"] OverweightObese
+        or ( exists [ConditionProblemsHealthConcerns: "Overweight or Obese"] OverweightObese
```

Comment:
This should be the union of conditions

Reply:
Based on the Authoring Patterns page (https://github.com/cqframework/CQL-Formatting-and-Usage-Wiki/wiki/Authoring-Patterns-QICore-v6.0.0), it seems that the presence of `prevalenceInterval` is associated with ConditionProblemsHealthConcerns. 

"_Problem list items and health concerns are typically documented with additional information about the condition such as prevalence period and clinical status... In particular, clinical status and prevalence period would only be expected to be present on problem list items and health concerns_." Am I interpreting this correctly?

Reply:
After reading that, I agree.